src/Index.tsx

```diff
@@ -5,8 +5,9 @@ import { saveItemInfosOnStateChange } from 'app/inventory/observers';
 import { loadCoreSettings } from 'app/manifest/actions';
 import { pollForBungieAlerts } from 'app/shell/alerts';
 import store from 'app/store/store';
+import { startStreamDeckConnection } from 'app/stream-deck/actions';
+import { $streamDeckFeature } from 'app/stream-deck/StreamDeckSettings/StreamDeckSettings';
```

Comment:
We do feature settings by adding to `$featureFlags` in `webpack.config.js` and in `global.d.ts`.

Reply:
Yeah I see that but didn't know which feature you add there or not. So meanwhile I tried to keep it simple